extensions/firefox/content/PdfJsNetwork.jsm

```diff
@@ -0,0 +1,261 @@
+/* Copyright 2012 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* globals Components, Services, dump */
+
+// NOTE: Be careful what goes in this file, as it is also used from the context
+// of the addon. So using warn/error in here will break the addon.
+
+'use strict';
+
+Components.utils.import('resource://gre/modules/Services.jsm');
+
+var EXPORTED_SYMBOLS = ['NetworkManager'];
+
+function log(aMsg) {
```

Comment:
Let's keep it just in case. I hope this module will be refactored and this might be helpful.